tools/ports/sdl2.py

```diff
@@ -5,9 +5,9 @@
 
 import os
 
-TAG = 'version_24'
-HASH = '5a8181acdcce29cdda7e7a4cc876602740f5b9deebd366ecec71ae15c4bbf1f352da4dd0e3c5e0ba8160709dda0270566d64a6cd3892da894463ecf8502836aa'
-SUBDIR = 'SDL2-' + TAG
+TAG = '5b646cd19e36e02c658fcbf792721b5bdce8491f'
```

Comment:
Is there some tag we could use here?  I wonder if upstream could mind tagging a release so that we could point to something here?

Reply:
Hmm, there will be a tag when 2.01.8 releases. Until then there isn't a released version with all the downstream changes. (Mostly missing timers without threads)

Reply:
OK, sgtm to use the tag for now then.  Perhaps add a TODO to update to 2.01.8 once it is released?